CHANGELOG.md

```diff
@@ -31,7 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Expose `tryExecuteAction` and `doLedgerAction` of the `Swap` class.
 
-## [0.10.0] - 2020-01-31
+## [0.10.0] - 2020-01-31®
```

Comment:
Why this R? 
Change is from 11 to 12, no...?

Reply:
It's a typo....

Reply:
ctrl-r: run tests
ctrl-option-r: open "Run" selection
option-r: ®
:)